### ISSUE TITLE
OIP 12 - Healthier food

### ISSUE THREAD

**Issue by leonardoalt** [ethdevberlin/OIPS#12]
I *think* there's a general consensus that we shouldn't have all that unhealthy food around in the office. Especially since the new person started buying things instead of @mvglasenapp , there's gummy bears, muffins and a lot more candy than before.
People will eat it anyway (myself included) since it's just there, it's easy and we don't have time.

Also brought up by @Marenz and @erak , re-starting discussion here.

**Comment by gumb0** [comment]
I agree, I'd be glad if we get rid of chocolate and candies

**Comment by ligi** [comment]
Fully support this one!

**Comment by leonardoalt** [comment]
Maybe we should do an (@axic) Apple move and push `getting rid of chocolate and candy` down our own throats ;)

Or at least ask everyone if they would mind that.

**Comment by LianaHus** [comment]
Agree. We can still buy good quality chocolate(not twix etc), nuts, maybe some dry fruits,  but everything else is simply bad for our health.

**Comment by axic** [comment]
> Maybe we should do an (@axic) Apple move and push getting rid of chocolate and candy down our own throats ;)

Do you want me to theatrically go and throw all the candy out the window?

**Comment by leonardoalt** [comment]
Yes. But maybe wait til someone else is there an can film?

**Comment by axic** [comment]
Also on that I'd propose this very controversial move: consider Charitea mate which tastes less sweet, is a smaller bottle (0.33 vs 0.5, e.g. doesnt force to drink all that sugar at once) and apparently has less sugar per ml.

**Comment by Marenz** [comment]
Or zero sugar mio cola! but we should conflate different issues here. This is purely about the candy.

**Comment by franzihei** [comment]
Hey there! Just found this issue and have to agree 100%! What was the outcome of this? Currently, we have a lot of very cheap baked goods (donuts, Berliner) at the office, alongside a big bowl with Twix, Snickers etc. I really don't mind a good croissant sometimes but these fatty things from the cheap bakers are not great, but super hard to resist when they are all laying there and there's no other option to choose from. Also, what happened to fruits from Kraut & Rübe (https://github.com/ethdevberlin/OIPS/issues/13#issuecomment-563913833)? The fruit we have at the office don't seem to be from there tbh? In addition to good fruit, I would recommend switching to healthy snacks like:
- dried fruits
- nuts
- oats (maybe even some oatmeal topping options)
- seaweed snacks
- yoghurt (?)
- healthy-ish granola bars (?) 

What do you guys think? I'm also super happy to facilitate / organize the orders at Rewe for this kind of stuff!! And also to buy fruits at Kraut & Rübe if it's too difficult to organize otherwise.
Edit: cc @renaynay :)

**Comment by mvglasenapp** [comment]
Thanks Franzi,

The person taking care of that just changed, now it's Agneska and previously it was Edyta. Agneska will need a little more introduction before everything works as before, but she already started quite well and I'm sure she can make it as we want.

So, regarding:
- "the very cheap baked goods": this is part of the pastry delivery that we usually have on Mondays, as a good start into the week. My feeling is that it's quite appreciated (in normal times, when the office is normally crowded) so I would opt to keep that. Aslo, we had croissants, Apfeltaschen and other things before, so let's sync to agree on which other pastries we would like Agneska to purchase. We could even try to find out which bakery has the best croissants (and we have a bunch of experts around the french corner..)

- twix etc: I will hide it again. To those who want to know the hiding place, just let me know ;)

- Kraut & Rube: right! It worked well with Edyta, so I will take care to get also Agneska doing so.

- healthy snacks: Sure, let's sync in the office to see which ones we would like to have next week for me to update Agneska. Then we will figure out which ones are successful or not for future weeks

- organising Rewe deliveries: we can also just mark down what we need on the black board and I'll take care of asking Agneska to purchase this. I had some bad experiences with ordering at Rewe i.e. it's offten delayed, it does not arrive on time, items are missing, a minimum order amount is needed... And it needs some time to be stored. So I believe that having Agneska taking care of this, is a very convenient solution for us.

**Comment by ligi** [comment]
Need to bring this up again. I just needed to throw away pastry as it was hard like a rock. And I hate to throw away food!
I think there should be no buying of pastry (at least) for the lockdown. And when I see the sentiment in here even longer. Also there is *a lot* of  Chocolate  again and not a lot of healthy stuff - IMHO also buying of (such) Chocolate should stop (even though it does not go bad - but it is unhealthy - main topic of this OIP) I think for the lockdown we should go with suggestions from @franzihei that do not go bad:
 * dried fruits
 * nuts
 * oats (maybe even some oatmeal topping options)
 * healthy-ish granola bars (?)

Also I think for now we should stop orderign banana - they also look as they go bad soon. Also they do not grow here.
